src/CommunityToolkit.Maui.Markup.UnitTests/BindableObjectExtensionsTests.cs

```diff
@@ -475,7 +475,7 @@ public void BindDefaultPropertyWithInlineTwoWayParameterizedConvertAndDefaults()
 					ArgumentNullException.ThrowIfNull(text);
 					ArgumentNullException.ThrowIfNull(repeat);
 
-					return text.Substring(0, text.Length / repeat.Value).Trim('\'');
+					return text[..(text.Length / repeat.Value)].Trim('\'');
```

Comment:
fancy

Reply:
I can't take any credit. Intellisense told me to do it 😆